packages/react-native-editor/__device-tests__/pages/editor-page.js

```diff
@@ -777,6 +786,29 @@ class EditorPage {
 	async sauceJobStatus( allPassed ) {
 		await this.driver.sauceJobStatus( allPassed );
 	}
+
+	// =========================
+	// Shortcode Block functions
+	// =========================
+
+	async getShortBlockTextInputAtPosition( blockName, position = 1 ) {
+		// iOS needs a click to get the text element
+		if ( ! isAndroid() ) {
+			const textBlockLocator = `(//XCUIElementTypeButton[contains(@name, "Shortcode Block. Row ${ position }")])`;
+
+			const textBlock = await waitForVisible(
+				this.driver,
+				textBlockLocator
+			);
+			await textBlock.click();
+		}
+
+		const blockLocator = isAndroid()
+			? `//android.view.ViewGroup[@content-desc="Shortcode Block. Row ${ position }"]/android.view.ViewGroup/android.view.ViewGroup/android.widget.EditText`
```

Comment:
As far as I tested locally, this locator can't be found when running the Android E2E tests locally. I re-run the tests several times and always fail when querying this element, @geriux have you encountered this issue too? 

**Error message:**
```
"//android.view.ViewGroup[@content-desc="Shortcode Block. Row 1"]/android.view.ViewGroup/android.view.ViewGroup/android.widget.EditText" is still not visible after 25 retries!
```

Reply:
I checked the XPath of the TextInput element of the Shortcode using the Appium inspector, and contains a small difference (note that `android.view.ViewGroup` has `[2]` at the end.):
```
//android.view.ViewGroup[@content-desc="Shortcode Block. Row 1"]/android.view.ViewGroup[2]/android.widget.EditText
```

In fact, I tried to use the path we're using in the test (i.e. `//android.view.ViewGroup[@content-desc="Shortcode Block. Row 1"]/android.view.ViewGroup/android.view.ViewGroup/android.widget.EditText`) and the Appium inspector can't find the element 🤔 .

Reply:
Interesting, just to double-check, do you have `English` set as the device language?

Reply:
Sorry, just saw your next message with the different locator, I'll investigate this.

Reply:
> Interesting, just to double-check, do you have English set as the device language?

Yeah, the device language is English.

> Sorry, just saw your next message with the different locator, I'll investigate this.

No worries 😊. I see that the E2E tests passed in the GB-mobile PR, so this might be an issue related to the emulator, I'm using `Pixel 3 (Android 9)` in case it helps.

Reply:
So I'm trying this locally and I get no errors, I can look for the element using the Appium inspector successfully searching for the `XPath`: 

```
//android.view.ViewGroup[@content-desc="Shortcode Block. Row 1"]/android.view.ViewGroup/android.view.ViewGroup/android.widget.EditText
```

I wonder why it doesn't work for you 🤔 

If I manually inspect the element I do see the XPath you mentioned above:

```
//android.view.ViewGroup[@content-desc="Shortcode Block. Row 1"]/android.view.ViewGroup[2]/android.view.ViewGroup/android.widget.EditText
```

Reply:
>I'm using Pixel 3 (Android 9) in case it helps.

I'll try that one

Reply:
> So I'm trying this locally and I get no errors, I can look for the element using the Appium inspector successfully searching for the `XPath`:
> 
> ```
> //android.view.ViewGroup[@content-desc="Shortcode Block. Row 1"]/android.view.ViewGroup/android.view.ViewGroup/android.widget.EditText
> ```
> 
> I wonder why it doesn't work for you 🤔
> 
> If I manually inspect the element I do see the XPath you mentioned above:
> 
> ```
> //android.view.ViewGroup[@content-desc="Shortcode Block. Row 1"]/android.view.ViewGroup[2]/android.view.ViewGroup/android.widget.EditText
> ```

Interesting, I see that the path is different for me. In mine, `EditText` has only one `ViewGroup` element and yours have two 🤔 :

The Xpath I'm getting:
```
//android.view.ViewGroup[@content-desc="Shortcode Block. Row 1"]/android.view.ViewGroup[2]/android.widget.EditText
```

Your Xpath:
```
//android.view.ViewGroup[@content-desc="Shortcode Block. Row 1"]/android.view.ViewGroup[2]/android.view.ViewGroup/android.widget.EditText
```

Reply:
In any case, if the CI jobs of the E2E tests are passing I wouldn't block the PR due to this issue.